test_sets/ideas_SetAudioStreamingIndicator_conditions_for_SDL_to_handle_related_cases.txt

```diff
@@ -57,4 +57,6 @@
 ./test_scripts/API/SetAudioStreamingIndicator/062_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_null_permission.lua Check_that_SDL_responds_with_DISALLOWED_when_navi_app_has_null_permissions
 ./test_scripts/API/SetAudioStreamingIndicator/063_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_null_permission.lua Check_that_SDL_responds_with_DISALLOWED_when_comm_app_has_null_permissions
 ./test_scripts/API/SetAudioStreamingIndicator/064_ATF_N_wrong_Json_SetAudioStreamingIndicator_INVALID_DATA.lua Check_that_SDL_responds_with_INVALID_DATA_when_RPC_SetAudioStreamingIndicator_comes_with_wrong_json_format_missing_:
-./test_scripts/API/SetAudioStreamingIndicator/065_ATF_N_HMI_responds_Invalid_Json_SetAudioStreamingIndicator.lua Check_that_SDL_responds_with_GENERIC_ERROR_when_HMI_responds_with_invalid_json
\ No newline at end of file
+./test_scripts/API/SetAudioStreamingIndicator/065_ATF_N_HMI_responds_Invalid_Json_SetAudioStreamingIndicator.lua Check_that_SDL_responds_with_GENERIC_ERROR_when_HMI_responds_with_invalid_json
+./test_scripts/API/SetAudioStreamingIndicator/066_ATF_P_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_true.lua Check_that_SDL_responds_with_IGNORED_of_second_request_before_HMI_responds_first_request_success_true
+./test_scripts/API/SetAudioStreamingIndicator/067_ATF_N_TC_SetAudioStreamingIndicator_media_app_sends_2_requests_HMI_replies_resultcode_success_false.lua Check_that_SDL_responds_with_IGNORED_of_second_request_before_HMI_responds_first_request_success_false
```

Comment:
@dboltovskyi, according to e-mail of Igor Kovalenko, for now empty lines should not be used.